Draft

brett6886 opened the pull request:
I deleted all the old files and added new templates, statics, and views. When the server is running, the control hub is at http://127.0.0.1:8000/androidprocessmanager/. Gotty browser shell is at http://127.0.0.1:8090/. Deleting any of the processes will also delete its child processes. The next steps will be to 1) create a thread that continuously checks the status of all processes (in case of a disconnect from the server), 2) create a system to deal with those process terminations, and 3) standardize the project to handle any number of scripts/processes that are added to the services folder.

brett6886 committed:
Deleted unused files, created browser-based control hub, and added working viewsin django

brett6886 committed:
deleted old, unused files and replaced with new draft views and files

brett6886 committed:
add new templates, statics, and views

swxdevops commented:
We really should be doing this to [sofwerx/docker-djangoforandroid](github.com/sofwerx/docker-djangoforandroid) instead.

ianblenke commented on _dockerignore:
```diff
@@ -0,0 +1,2 @@
+.git
```
I'm not sure what a `_dockerignore` file is. Did you mean a `.dockerignore` file?

ianblenke reviewed (commented):
It would have been better to do a `git mv example  androidprocessmanager` do carry through the git history and avoid all of the add/remove changes for things that were simply renamed.

The `*.pyc` files really should never be added to the git repo. This is one of the things that a `.gitignore` file should have, to prevent this from happenning. Can you remove these files from your branch? Thanks!

brett6886 committed:
deleted *.pyc files from __pycache__

brett6886 committed:
Detailed README and cleanup

brett6886 committed:
Deleted unecessary _dockerignore file

brett6886 committed:
deleted .pyc files

brett6886 committed:
Added .gitignore to ignore *.pyc files

brett6886 committed:
Better Comments and Fixed Errors

Organized and rewrote all comments in django views and javascript file. Also fixed the error caused by django sending back incorrect zombie PIDs. It was just a matter of updating the dictionary of running processes.

brett6886 committed:
more detailed comments

added more comments to javascript and html files

brett6886 committed:
Replaced runProcess 1 & 2 with runProcess

I increased the versatility of the javascript file so it can handle any number of buttons/processes. I also added additional comments
